src/minefield/game.tests.cpp

```diff
@@ -1,16 +1,79 @@
 #include <gtest/gtest.h>
 #include <minefield/utils.h>
+#include <minefield/game_states.h>
 
 namespace minefield::game::tests
 {
-TEST(enteringBoardMeasures, should)
+class GameTestSuit : public ::testing::Test
 {
-    unsigned int min = 10;
-    unsigned int max = 15;
-    unsigned int width = Utils::enterNumberInRange("message", 10, 15);
-    bool widthIsBetweenRange = width < min && width > max;
-    EXPECT_TRUE(widthIsBetweenRange);
-}   
+protected:
+    void SetUp() override
+    {
+    }
 
+    void TearDown() override
+    {
+    }
```

Comment:
#should remove these implementations if they'll be empty - they're not pure virtuals in the base class